handlers/meme.js

```diff
@@ -33,7 +33,7 @@ async function meme(message) {
     return;
   }
 
-  
+
```

Comment:
lol

Reply:
I think my editor did it automatically haha